pkg/infra/usagestats/usage_stats.go

```diff
@@ -99,6 +99,28 @@ func (uss *UsageStatsService) GetUsageReport(ctx context.Context) (UsageReport,
 		} else {
 			dsOtherCount += dsStat.Count
 		}
+
+		if dsStat.Type == models.DS_ES {
+			dsSettings := models.GetESJSONDataQuery{}
+			if err := uss.Bus.Dispatch(&dsSettings); err != nil {
+				metricsLogger.Error("Failed to get elasticsearch json data", "error", err)
+				return report, err
+			}
+
+			for _, data := range dsSettings.Result {
+				var m models.ESJSONData
+
+				if err := json.Unmarshal([]byte(data.JsonData), &m); err != nil {
+					return report, err
+				}
+
+				statName := fmt.Sprintf("stats.ds.elasticsearch.v%d.count", m.ESVersion)
```

Comment:
this will push metrics like `stats.ds.elasticsearch.v56.count` or `stats.ds.elasticsearch.v70.count` alongside `stats.ds.elasticsearch.count` in order to not disrupt any existing metrics we already have.